src/main/kotlin/com/djvk/sarPdfParser/exceptions/FileProcessingException.kt

```diff
@@ -1,7 +1,4 @@
 package com.djvk.sarPdfParser.exceptions
 
 class FileProcessingException(private val fileName: String, exception: Exception): Exception(exception.message, exception.cause) {
-    override fun getLocalizedMessage(): String {
```

Comment:
Why remove this?

Reply:
I was removing `"Filename: $fileName"` from the string (based on CLI output and what shows up in the errors.csv, the filename is always included in the error message anyway), when I realized that the body of this function would just be `super.getLocalizedMessage()`, so I just removed the whole function.